phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1846,6 +1848,32 @@ public function format_orderedlist($open, $name, $attrs, $props) {
         return '</ol>';
     }
 
+    /* Support for key inputs is coded like junk */
+    public function format_keycap($open, $name, $attrs, $props) {
+        if ($open) {
+            $content = '';
+            if ($props['sibling']) {
+                $content = ' + ';
```

Comment:
```suggestion
                $content = '+';
```

Nit: suggest not adding spaces around the `+` character, so that the rendered HTML can look like `CTRL+C` rather than `CTRL + C`